### DISCUSSION TITLE
Hide menu bar?

### DISCUSSION THREAD

**Discussion by mhwombat** [orbitalquark/textadept#229] [Q&A] [answered]
Now that I have the [textadept-hydra plugin](https://github.com/mhwombat/textadept-hydra), I don't need the top menu bar at all. Is there a way to hide it?

**Comment by orbitalquark** [comment, marked as answer]
In your *~/.textadept/init.lua*:

  if not OSX then events.connect(events.INITIALIZED, function() textadept.menu.menubar = nil end) end

**Comment by mhwombat** [comment]
Beautiful!

**Comment by orbitalquark** [comment]
In your *~/.textadept/init.lua*:

  if not OSX then events.connect(events.INITIALIZED, function() textadept.menu.menubar = nil end) end
